tests/ci/profile_handler.go

```diff
@@ -198,26 +258,35 @@ func GenerateClusterCreationArgsByProfile(profile *Profile) (clusterArgs *EXE.Cl
 	return clusterArgs, profile.ManifestsDIR, err
 }
 
-func LoadProfileYamlFile() *Profile {
-	profileEnv := GetEnvWithDefault(CON.RhcsClusterProfileENV, "rosa-sts-pl")
-	if profileEnv == "" {
-		panic(fmt.Errorf("ENV Variable RHCS_PROFILE_ENV is empty, please make sure you set the env value"))
-	}
+func LoadProfileYamlFile(profileName string) *Profile {
 	filename := GetYAMLProfileFile(CON.TFYAMLProfile)
-	p := HELPER.GetProfile(profileEnv, filename)
+	p := HELPER.GetProfile(profileName, filename)
 	fmt.Println(p.Cluster)
 	profile := Profile{
-		Name: profileEnv,
+		Name: profileName,
 	}
 	err := HELPER.MapStructure(p.Cluster, &profile)
+	if profile.ManifestsDIR == "" {
+		profile.ManifestsDIR = CON.ROSAClassic
+	}
 	Expect(err).ToNot(HaveOccurred())
 	return &profile
 }
 
-func CreateRHCSClusterByProfile(profile *Profile, creationArgs *EXE.ClusterCreationArgs, manifests_dir string) (string, error) {
+func LoadProfileYamlFileByENV() *Profile {
+	profileEnv := os.Getenv("CLUSTER_PROFILE")
+	if profileEnv == "" {
+		panic(fmt.Errorf("ENV Variable RHCS_PROFILE_ENV is empty, please make sure you set the env value"))
+	}
```

Comment:
actually - why are we using here `CLUSTER_PROFILE` instead of `RHCS_PROFILE_ENV`?

Reply:
Removed RHCS_PROFILE_ENV
instead using CLUSTER_PROFILE to keep consistent between testing components.
Will updating in future if required. :)

Reply:
ack :)